Resources/Locale/en-US/mass-media/news-discord.ftl

```diff
@@ -0,0 +1,5 @@
+news-discord-footer =
+    Author: {$author}
+    Server: {$server}
+    Round: #{$round}
```

Comment:
i didn't even know discord didn't strip newlines from embed footers, but this should be something like
```suggestion
news-discord-footer = Author: {$author} | Server: {$server} | Round: #{$round}
```
to avoid an oversized footer

also, author should probably be conveyed by the dedicated embed field for it
in which case
```suggestion
news-discord-footer = Server: {$server} | Round: #{$round}
```